lib/net/java/dev/spellcast/utilities/ChatBuffer.java

```diff
@@ -136,8 +136,16 @@ public JScrollPane addDisplay( final JEditorPane displayPane )
 	 * Sets the log file used to actively record messages that are being stored in the buffer.
 	 */
 
-	public void setLogFile( final File f )
-	{
+	public void setLogFile( final File f ) {
```

Comment:
As a general rule we treat Java source in /lib/net as being someone else's code and we do not modify it in case they ever update.  On that alone I would oppose theses changes.  If there is a reason for them and, better yet, a failing test I'd be more willing to relax that rule or at least see if the changes could be done in the calling code instead.

Reply:
Reverted the changes to using ChatBuffer